docs/databases/swift-kuery.html

```diff
@@ -252,237 +229,150 @@ <h2 class="heading-2"><span class="blue-text">Step 5 (Optional):</span> Test sav
     "title": "A Game of Thrones",
     "price": 14.99,
     "genre": "Fantasy"
-  }'</code></pre>
-      <p>Then using the PostgreSQL CLI we can query the database to see if our data was stored.</p>
-      <div class="info">
-        <p>You can start the postgreSQL CLI by running the following in a terminal:</p>
-        <pre><code>psql bookstoredb</code></pre>
-      </div>
-      <p>In a terminal which is running the `psql` command run the following:</p>
-      <pre><code>SELECT * FROM "Titles";</code></pre>
+}'</code></pre>
+    <p>This will make a POST request to the server and we should be returned our book in JSON format:</p>
+    <pre><code>{"id":0,"title":"A Game of Thrones","price":14.99,"genre":"Fantasy"}</code></pre>
+    <div class="info">
+      <p>The following steps are for PostgreSQL. The commands will be different if you used a different database.</p>
+    </div>
+      <p>Then using the PostgreSQL CLI we can query the database to see if our data was saved.</p>
+      <p>Start the postgreSQL CLI for our database by running the following in a terminal:</p>
+      <pre><code>psql bookstoredb</code></pre>
+      <p>In a terminal which is running the `psql` command, run the following:</p>
+      <pre><code>SELECT * FROM "BookTable";</code></pre>
       <p>You should see the following printed in the terminal:</p>
       <pre><code> id |       title       | price |  genre
   ----+-------------------+-------+---------
    0  | A Game of Thrones | 14.99 | Fantasy
   (1 row)</code></pre>
       <p>This proves our POST work is working as we'd expect.</p>
-      <p>However the best way to test this would be to actually query the database for the results rather than using the terminal.</p>
+      <p>However the best way to test this would be to actually query the database in Swift rather than using the terminal.</p>
```

Comment:
How about "A better way to test this, as we're writing Swift code, would be to query the database directly from our Swift code rather than using Terminal."